__tests__/app.test.js

```diff
@@ -166,6 +166,114 @@ describe('/api/reviews', () => {
+                expect(typeof review === 'object');
+            })
+            res.body.forEach((review) => {
+                expect(review).toEqual({
+                    review_id: expect.any(Number),
+                    title: expect.any(String),
+                    category: expect.any(String),
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    comment_count: expect.any(Number)
+                })
+            })
+            expect(res.body).toBeSortedBy("created_at", { descending: false });
+
+        });
+    });
+
+    test('GET - 200: respond with an array of review objects sorted in ASC order of votes.', () => {
+        return request(app)
+        .get('/api/reviews?sort_by=votes&order=ASC')
+        .expect(200)
+        .then((res) => {
+            expect(res.body).toEqual(expect.any(Array))
+            res.body.forEach((review) => {
+                expect(typeof review === 'object');
+            })
+            res.body.forEach((review) => {
+                expect(review).toEqual({
+                    review_id: expect.any(Number),
+                    title: expect.any(String),
+                    category: expect.any(String),
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    comment_count: expect.any(Number)
+                })
+            })
+            expect(res.body).toBeSortedBy("votes", { descending: false });
+
+        });
+    });
+
+    test('GET - 200: respond with an array of review objects sorted in ASC order of votes with a specific category.', () => {
+        return request(app)
+        .get('/api/reviews?sort_by=votes&order=ASC&category=social deduction')
+        .expect(200)
+        .then((res) => {
+            expect(res.body).toEqual(expect.any(Array))
+            res.body.forEach((review) => {
+                expect(typeof review === 'object');
+            })
+            res.body.forEach((review) => {
+                expect(review).toEqual({
+                    review_id: expect.any(Number),
+                    title: expect.any(String),
+                    category: 'social deduction',
+                    designer: expect.any(String),
+                    owner: expect.any(String),
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    comment_count: expect.any(Number)
+                })
+            })
+            expect(res.body).toBeSortedBy("votes", { descending: false });
+
+        });
+    });
+
```

Comment:
I'd add a test for 
- valid category, but if it has no reviews (200)
- non existent category (404)